crates/cairo-lint-core/tests/test_files/erasing_operations/erasing_operations

```diff
@@ -0,0 +1,111 @@
+//! > Multiplication by zero
```

Comment:
can you prefix every variable with `_` so we don't have the unused variable annoying diagnostic ?